Apps/W1/HybridGP/test/src/GPItemTests.codeunit.al

```diff
@@ -46,26 +47,38 @@ codeunit 139662 "GP Item Tests"
             Migrate(GPItem);
         until GPItem.Next() = 0;
 
-        // [THEN] A Item is created for all staging table entries
-        Assert.RecordCount(Item, GPItem.Count());
-        Assert.AreEqual(GPItem.Count(), HelperFunctions.GetNumberOfItems(), 'Wrong number of Items calculated');
+        HelperFunctions.CreatePostMigrationData();
+
+        // [THEN] An Item is created for all configured staging table entries
+        Assert.RecordCount(Item, 7);
```

Comment:
Minor, lets not hardcode 7, could we have a variable at least close where the data is created? The ideal solution would be to calculate it from some data, if it is possible.